packages/core/test/popover/popoverTests.tsx

```diff
@@ -297,10 +355,9 @@ describe("<Popover>", () => {
                     interactionKind: "hover",
                     usePortal: true,
                 });
-                const targetElement = wrapper.findClass(Classes.POPOVER_TARGET);
-                targetElement.simulate("focus");
-                targetElement.simulate("blur");
-                assert.isTrue(wrapper.state("isOpen"));
+                wrapper.simulateTarget("focus");
```

Comment:
this should probably be:
```suggestion
                wrapper.focusTargetButton();
```